### ISSUE TITLE
"Blank Page" after accepting the self-signed cert

### ISSUE THREAD

**Issue by cjrollo** [AlexGaspar/docker-fbctf#2]
single host, running via "docker-compose up". (on a mac, hence the 192.168.99.100 address)
after doing the self-signed cert dance in FF, I get a blank page? 

fbctf_1      | /var/www/fbctf
fbctf_1      | Database already created... skipping creation...
fbctf_1      | 2016/05/19 07:16:22 [warn] 1#0: "ssl_stapling" ignored, issuer certificate not found
fbctf_1      | nginx: [warn] "ssl_stapling" ignored, issuer certificate not found
fbctf_1      | 192.168.99.1 - - [19/May/2016:07:18:00 +0000] "GET / HTTP/1.1" 500 5 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.11; rv:46.0) Gecko/20100101 Firefox/46.0"
fbctf_1      | 2016/05/19 07:18:00 [error] 43#0: *7 open() "/var/www/fbctf/src/favicon.ico" failed (2: No such file or directory), client: 192.168.99.1, server: , request: "GET /favicon.ico HTTP/1.1", host: "192.168.99.100"
fbctf_1      | 192.168.99.1 - - [19/May/2016:07:18:00 +0000] "GET /favicon.ico HTTP/1.1" 404 48 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.11; rv:46.0) Gecko/20100101 Firefox/46.0"
fbctf_1      | 2016/05/19 07:18:00 [error] 43#0: *7 open() "/var/www/fbctf/src/favicon.ico" failed (2: No such file or directory), client: 192.168.99.1, server: , request: "GET /favicon.ico HTTP/1.1", host: "192.168.99.100"
fbctf_1      | 192.168.99.1 - - [19/May/2016:07:18:00 +0000] "GET /favicon.ico HTTP/1.1" 404 48 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.11; rv:46.0) Gecko/20100101 Firefox/46.0"

**Comment by cjrollo** [comment]
$ docker logs 689f975cc0f4
Generating self-signed certificate...
Generating RSA private key, 2048 bit long modulus
...................+++
.................................................................................................................+++
e is 65537 (0x10001)
writing RSA key
Signature ok
subject=/C=UK/ST=NRW/L=London/O=My Inc/OU=DevOps/CN=www.example.com/emailAddress=dev@example.com
Getting Private key
Generating DH parameters, 2048 bit long safe prime, generator 2
This is going to take a long time
.........................................+.........................................................................................+.......+.........................................+..............................+..............................+..........................+.........++_++_
/var/www/fbctf
Database already created... skipping creation...
2016/05/19 07:23:02 [warn] 1#0: "ssl_stapling" ignored, issuer certificate not found
nginx: [warn] "ssl_stapling" ignored, issuer certificate not found
192.168.99.1 - - [19/May/2016:07:25:50 +0000] "GET / HTTP/1.1" 301 178 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.11; rv:46.0) Gecko/20100101 Firefox/46.0"

**Comment by AlexGaspar** [comment]
Thanks for reporting this issue :)

It should be fixed, could you try : `docker pull alexgaspar/fbctf:latest` and then `docker-compose up --force-recreate`

**Comment by cjrollo** [comment]
Site works a charm now! Thanks for the work. Did notice some errors when spinning up the mysql server:

$ docker-compose logs
Attaching to dockerfbctf_fbctf_1, dockerfbctf_mysql_1, dockerfbctf_memcached_1
fbctf_1      | Generating self-signed certificate...
fbctf_1      | Generating RSA private key, 2048 bit long modulus
fbctf_1      | ..........................................+++
fbctf_1      | ...........+++
fbctf_1      | e is 65537 (0x10001)
fbctf_1      | writing RSA key
fbctf_1      | Signature ok
fbctf_1      | subject=/C=UK/ST=NRW/L=London/O=My Inc/OU=DevOps/CN=www.example.com/emailAddress=dev@example.com
fbctf_1      | Getting Private key
fbctf_1      | Generating DH parameters, 2048 bit long safe prime, generator 2
fbctf_1      | This is going to take a long time
fbctf_1      | _++_
(deleted a bunch of cruft)
fbctf_1      | [+] Creating DB - fbctf
fbctf_1      | ERROR 1007 (HY000) at line 1: Can't create database 'fbctf'; database exists
fbctf_1      | [+] Importing schema...
fbctf_1      | [+] Importing countries...
fbctf_1      | [+] Importing logos...
fbctf_1      | [+] Creating user...
fbctf_1      | ERROR 1227 (42000) at line 1: Access denied; you need (at least one of) the CREATE USER privilege(s) for this operation
fbctf_1      | ERROR 1044 (42000) at line 1: Access denied for user 'fbctf'@'%' to database 'fbctf'
fbctf_1      | ERROR 1227 (42000) at line 1: Access denied; you need (at least one of) the RELOAD privilege(s) for this operation
fbctf_1      | [+] DB Connection file
fbctf_1      | [+] Adding default admin user
fbctf_1      | [+] The password for admin is: bf8111d29cac9e6293a0d4bc30aa4208
fbctf_1      | 2016/05/19 20:32:15 [warn] 1#0: "ssl_stapling" ignored, issuer certificate not found
fbctf_1      | nginx: [warn] "ssl_stapling" ignored, issuer certificate not found
mysql_1      | 160519 20:29:49 [Note] mysqld (mysqld 5.5.49) starting as process 1 ...
mysql_1      | 160519 20:29:49 [Note] Plugin 'FEDERATED' is disabled.
mysql_1      | 160519 20:29:49 InnoDB: The InnoDB memory heap is disabled
mysql_1      | 160519 20:29:49 InnoDB: Mutexes and rw_locks use GCC atomic builtins
mysql_1      | 160519 20:29:49 InnoDB: Compressed tables use zlib 1.2.3
mysql_1      | 160519 20:29:49 InnoDB: Using Linux native AIO
mysql_1      | 160519 20:29:49 InnoDB: Initializing buffer pool, size = 128.0M
mysql_1      | 160519 20:29:49 InnoDB: Completed initialization of buffer pool
mysql_1      | 160519 20:29:49 InnoDB: highest supported file format is Barracuda.
mysql_1      | 160519 20:29:50  InnoDB: Waiting for the background threads to start
mysql_1      | 160519 20:29:51 InnoDB: 5.5.49 started; log sequence number 1595675
mysql_1      | 160519 20:29:51 [Note] Server hostname (bind-address): '0.0.0.0'; port: 3306
mysql_1      | 160519 20:29:51 [Note]   - '0.0.0.0' resolves to '0.0.0.0';
mysql_1      | 160519 20:29:51 [Note] Server socket created on IP: '0.0.0.0'.
mysql_1      | 160519 20:29:51 [Warning] 'proxies_priv' entry '@ root@e014ec697951' ignored in --skip-name-resolve mode.
mysql_1      | 160519 20:29:51 [Note] Event Scheduler: Loaded 0 events
mysql_1      | 160519 20:29:51 [Note] mysqld: ready for connections.
mysql_1      | Version: '5.5.49'  socket: '/tmp/mysql.sock'  port: 3306  MySQL Community Server (GPL)

**Comment by AlexGaspar** [comment]
Not sure these are important errors. 

The first one occur because the database is already created, no big deal there.
The second and the third, are the script trying to set permission to fbctf to be able use it from anywhere (`@'%'`) and since fbctf doesn't have permissions to set this kind of permissions it's denied. 

So we could run this script as root or just ignore those errors since fbctf already got all privileges from anywhere :) 

```
mysql> SHOW GRANTS for fbctf; 
+------------------------------------------------------------------------------------------------------+
| Grants for fbctf@%                                                                                   |
+------------------------------------------------------------------------------------------------------+
| GRANT USAGE ON *.* TO 'fbctf'@'%' IDENTIFIED BY PASSWORD '*5F27A5A4B9E8998B7786C4AFB1B164E5DADC82C4' |
| GRANT ALL PRIVILEGES ON `fbctf`.* TO 'fbctf'@'%'                                                     |
+------------------------------------------------------------------------------------------------------+
```
